certified-connectors/Easyvista_Service._Manager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,13162 @@
+                  "description": "AVAILABLE_FIELD_1",
+                  "title": "AVAILABLE_FIELD_1",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "AVAILABLE_FIELD_2": {
+                  "description": "AVAILABLE_FIELD_2",
+                  "title": "AVAILABLE_FIELD_2",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "AVAILABLE_FIELD_3": {
+                  "description": "AVAILABLE_FIELD_3",
+                  "title": "AVAILABLE_FIELD_3",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "AVAILABLE_FIELD_4": {
+                  "description": "AVAILABLE_FIELD_4",
+                  "title": "AVAILABLE_FIELD_4",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "AVAILABLE_FIELD_5": {
+                  "description": "AVAILABLE_FIELD_5",
+                  "title": "AVAILABLE_FIELD_5",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "AVAILABLE_FIELD_6": {
+                  "description": "AVAILABLE_FIELD_6",
+                  "title": "AVAILABLE_FIELD_6",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "BEGIN_OF_CONTRACT": {
+                  "description": "BEGIN_OF_CONTRACT",
+                  "format": "date-time",
+                  "title": "BEGIN_OF_CONTRACT",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "CELLULAR_NUMBER": {
+                  "description": "CELLULAR_NUMBER",
+                  "title": "CELLULAR_NUMBER",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "CHAT_LOGIN": {
+                  "description": "CHAT_LOGIN",
+                  "title": "CHAT_LOGIN",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "CIVIL_STATUS_ID": {
+                  "description": "CIVIL_STATUS_ID",
+                  "title": "CIVIL_STATUS_ID",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "COMMENT_EMPLOYEE": {
+                  "description": "COMMENT_EMPLOYEE",
+                  "properties": {
+                    "HREF": {
+                      "description": "COMMENT_EMPLOYEE_HREF",
+                      "title": "COMMENT_EMPLOYEE_HREF",
+                      "type": "string",
+                      "x-ms-visibility": ""
+                    }
+                  },
+                  "type": "object"
+                },
+                "COST_PER_HOUR": {
+                  "description": "COST_PER_HOUR",
+                  "title": "COST_PER_HOUR
```

Comment:
@ezv-compteintegration at the end of the file, please add the "x-ms-connector-metadata". Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json).
```
  "x-ms-connector-metadata": [
    {
      "propertyName": "Website",
      "propertyValue": ""
    },
    {
      "propertyName": "Privacy policy",
      "propertyValue": ""
    },
    {
      "propertyName": "Categories",
      "propertyValue": ""
    }
  ]
```

Reply:
@Amjed-Ayoub 

I modified the whole apiDefinition.swagger.json file

Reply:
Hello @ezv-compteintegration,

Thank you so much for getting back to me. Unfortunately the PR is still having a lot of errors.
![image](https://github.com/microsoft/PowerPlatformConnectors/assets/97551789/3f0763e5-3fb5-4c98-9cb1-cc7007484bce)

I would be happy to schedule a quick call with you to discuss this in detail. May I ask you please to send us an email to our Connector Partner Mgmt Team <ConnectorPartnerMgmtTeam@service.microsoft.com>, and I will make sure to respond quickly. 

Thank you so much again.

Reply:
Hello,
yes we can. In fact, it's my colleague who make all the changes but now he is not here.
Thanks, my english is not very well but we can try

Reply:
@Amjed-Ayoub file modified, now no errors ;)